ansible/roles/prosody/files/mod_muc_webhooks.lua

```diff
@@ -227,6 +229,11 @@ function handle_occupant_access(event, event_type)
         breakout_room_id = jid_split(room.jid);
     end
 
+    local is_lobby = util.has_prefix(select(2, jid_split(room.jid)), LOBBY_PREFIX);
```

Comment:
You can do just has_prefix(jid.host(room.jid), ... )

Reply:
Hum whether a room is lobby, you can recognize it by checking is there a main_room set. 

https://github.com/jitsi/jitsi-meet/blob/22c6b72a7540c4bd3afd2d8362a218d07ce4a6ca/resources/prosody-plugins/mod_muc_lobby_rooms.lua#L207

It's easier and quicker than comparing the strings.